packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -498,8 +498,6 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
 
   bool get hasScrolledBody {
     for (_NestedScrollPosition position in _innerPositions) {
-      // TODO(chunhtai): Replace null check with assert once
-      // https://github.com/flutter/flutter/issues/31195 is fixed.
       if (
         position.minScrollExtent != null &&
```

Comment:
I intend to keep this check as there might be other corner case that hit this, but i cannot think of any now.

Reply:
Turn these into asserts then?